tutorials/forest-vegetation-condition/index.md

```diff
@@ -0,0 +1,131 @@
+
+## Context
+
+Forests around the world are in crisis. Rapidly expanding global human footprint and climate change have caused extensive deforestation, and have left remnant forests fragmented and significantly altered in condition. Yet, these remaining forests harbour a dazzling array of unique ecosystems and an overwhelming majority of our planet’s current terrestrial biodiversity, thanks to in-situ conservation efforts including those by local and indigenous communities. Valuing these forests and ensuring their continued conservation depends on our understanding their long term dynamics and responses to dominant anthropogenic and climatic drivers. This tutorial illustrates the use of Earth Engine to investigate forest vegetation condition over time.
+
+## Rationale
+
+Remotely sensed indices such as enhanced vegetation index (EVI), normalized difference vegetation index (NDVI) and normalized difference water index (NDWI) are widely used to estimate vegetation status from satellite imagery. EVI and NDVI estimate vegetation chlorophyll content while NDWI estimates vegetation moisture content. All these indices can be derived from free public imagery of satellites like Landsat and MODIS, available in the Earth Engine data catalog. This tutorial uses MODIS EVI data from the [MOD13Q1.006 dataset](https://developers.google.com/earth-engine/datasets/catalog/MODIS_006_MOD13Q1) for the period from year 2000 to 2019.
+
+Bandipur and Nagarhole national parks are located in the Mysore-Malenad landscape of the Western Ghats in southern India, and are home to diverse habitats ranging from dry deciduous scrub forests to evergreen forests. Forest vegetation in this region is typically under most stress during the dry summer months, and hence, tracking annual changes during this period can be useful from a conservation and management perspective. The vegetation turns most green and abundant during the rainy monsoon months, but also becomes hard to observe from optical satellite imagery due to the accompanying heavy cloud cover.
+
+## Approach
+
+### Import images, features
+
+Import the MODIS 250m/pixel 16-day composite vegetation indices dataset, and select the EVI band. Load boundaries of Bandipur and Nagarhole national parks.
+```js
+// get MODIS 250m EVI data 
+var mod13 = ee.ImageCollection('MODIS/006/MOD13Q1')
+  .select('EVI');
+// get features of the forest national parks
+var nps = ee.FeatureCollection('users/pradeepkoulgi/IndiaTRCondition/MainlandPAs')
+  .filter(ee.Filter.inList('NAME', ['Bandipur', 'Nagarhole']));
+```
+### Compute annual summertime composites 
+
+Build an image collection with an image for each year from 2000 to 2019, where the image is the maximum EVI in that year's summer months. This is our measure of the status of the vegetation for each year. Also add the year as a band, in preparation for linear trend analysis.
```

Comment:
If I understand correctly, the collection will contain multiple images, not just a single image; please reword accordingly.

Reply:
Done